Sources/App/Views/ReadyForSwift6/ReadyForSwift6Show+View.swift

```diff
@@ -38,17 +38,74 @@ extension ReadyForSwift6Show {
-                .p("To measure compatibility with Swift 6 across packages in the index, we are tracking compatibility across a set of packages under active development where they have at least one git commit in the past 12 months. The charts below visualise the results of our testing."),
+                .p("Swift 6 brings with it the ability to check your code for concurrency and data race issues. If switched on, the Swift compiler will produce errors where you could have data races."),
+                .p(.text("For help migrating your code, see the "),
+                   .a(
+                    .href("https://example.com"),
+                    .text("Swift 6 language mode migration guide")
+                   ),
+                   .text(" or the "),
+                   .a(
+                    .href("https://example.com"),
+                    .text("Swift 6 release blog post")
+                   )),
+                .p("To track the progress of the Swift package ecosystem, the Swift Package Index is running regular package compatibility checks across all packages in the index."),
                 .h3("Total packages compatible with Swift 6"),
-                .p("This chart shows the total number of packages that will compile with  Swift 6:"),
-                model.readyForSwift6Chart(identifier: "rfs6-packages"),
+                model.readyForSwift6Chart(kind: .compatiblePackages),
                 .h3("Total Swift 6 concurrency errors"),
-                .p("This chart shows the total number of Swift concurrency errors across the entire selection of testing packages:"),
-//                model.readyForSwift6Chart(chartIdentifier: "rfs6-errors"),
-                .h3("List of compatible packages"),
-                .p("Here are all the compatible packages!")
+                model.readyForSwift6Chart(kind: .totalErrors),
+                .h3("Frequently asked questions"),
+                .p(
+                    .strong(.text("Q: ")),
+                    .text("What does “compatible” mean in the chart of compatible packages?")
+                ),
+                .p(
+                    .strong(.text("A: ")),
+                    .text("We define compatibility in the same way we do on package pages. If any build of the package completes successfully on any of our tested platforms (macOS via SwiftPM, macOS via XcodeBuild, iOS, visionOS, watchOS, tvOS, or Linux) then that build is deemed compatible with the Swift version.")
+                ),
+                .hr(
+                    .class("minor")
+                ),
+                .p(
+                    .strong(.text("Q: ")),
+                    .text("What does “total concurrency errors” mean?")
+                ),
+                .p(
+                    .strong(.text("A: ")),
+                    .text("Sven: Can you replace this with a short description of these errors, please?")
```

Comment:
@finestructure I figured you'd do a better job with this description. Could you add it when you have a second please.